docs/source/developers/benchmarks.rst

```diff
@@ -33,22 +33,23 @@ The benchmark suites can be run with the ``benchmark run`` sub-command.
 
 .. code-block:: shell
 
-  # Run benchmarks in the current git workspace
-  archery benchmark run
-  # Storing the results in a file
-  archery benchmark run --output=run.json
+   # Run benchmarks in the current git workspace
```

Comment:
Hmm, strange that this mattered ..

Reply:
It didn't, but it's nicer to have consistent indents.